### ISSUE TITLE
Add {ARVAULT}: {ArVault} on {Arbitrum}

### ISSUE THREAD

**Issue by arvault-on-arbitrum** [sushiswap/default-token-list#253]
- [ x ] I understand that token listing is not required to use the SushiSwap Interface with a token.
- [ x ] I understand that filing an issue or adding liquidity does not guarantee addition to the Sushi default token list.
- [ x ] I will not ping the Discord about this listing request.

**Please provide the following information for your token.**

Network: Arbitrum
Token Address (checkSum): 0xa6219b4bf4b861a2b1c02da43b2af266186edc04
Token Name (from contract): ArVault
Token Decimals (from contract): 9
Token Symbol (from contract): ARVAULT
SushiSwap V2 Pair Address of Token: https://analytics-arbitrum.sushi.com/pairs/0x5b772b00cb6b95c4501e4be75ce7ddd6cb625320

Link to sushiswap/assets PR: https://github.com/sushiswap/assets/pull/236
Link to the official homepage of token: https://arvault.xyz/
Link to CoinMarketCap or CoinGecko page of token: Requested
Link to official Twitter account: https://twitter.com/vaultfarms

**Comment by arvaultxyz** [comment]
Indeed, arvaulties are a sizable community (1mm+ in forever locked liquidity, 300 holders). Thanks for starting this issue/PR @arvault-on-arbitrum!

**Comment by arvaultxyz** [comment]
Top 10 token on sushiswap arbitrum analytics too!
